# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1272539648 14400
#      Thu Apr 29 07:14:08 2010 -0400
# Node ID 083e4f6143fe89d4c265a9ee8e71644de9b9b917
# Parent  b6364a88de06d74f44fa0e813985eb03b7076d8f
__errplot__.m: Fix bug for boxxy errorbar.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,11 +1,12 @@
 2010-04-29  Ben Abbott <bpabbott@mac.com>
 
-	* plot/errorbar.m: Add demoss.
+	* plot/private/__errplot__.m: Fix bug for boxxy errorbar.
+	* plot/errorbar.m: Add demos.
 
 2010-04-27  David Bateman  <dbateman@free.fr>
 
 	* plot/private/__errplot__.m: Update the marker in a seperate
 	listener function.
 
 2010-04-27  Rik <octave@nomad.inbox5.com>
 
diff --git a/scripts/plot/private/__errplot__.m b/scripts/plot/private/__errplot__.m
--- a/scripts/plot/private/__errplot__.m
+++ b/scripts/plot/private/__errplot__.m
@@ -45,17 +45,17 @@ function h = __errplot__ (fstr, p, a1, a
     elseif (strcmp (fmt.errorstyle, ">"))
       ifmt = "xerr";
     elseif (strcmp (fmt.errorstyle, "~>"))
       ifmt = "xyerr";
     elseif (strcmp (fmt.errorstyle, "#"))
       ifmt = "box";
     elseif (strcmp (fmt.errorstyle, "#~"))
       ifmt = "boxy";
-    elseif (strcmp (fmt.linestyle, "#~>"))
+    elseif (strcmp (fmt.errorstyle, "#~>"))
       ifmt = "boxxy";
     else
       ifmt = "~";
     endif
 
     hg = hggroup ("parent", p);
     h = [h; hg];
     args = __add_datasource__ ("__errplot__", hg, 
